ios/MusicControlManager.m

```diff
@@ -231,36 +236,41 @@ - (void)sendEvent:(NSString*)event {
                        body:@{@"name": event}];
 }
 
-- (void)updateArtworkIfNeeded:(id)artwork
-{
-    NSString *url = nil;
-    if (artwork) {
-        if ([artwork isKindOfClass:[NSString class]]) {
-             url = artwork;
-        } else if ([[artwork valueForKey: @"uri"] isKindOfClass:[NSString class]]) {
-             url = [artwork valueForKey: @"uri"];
-        }
-    }
+- (NSString*)getArtworkUrl:(NSString*)artwork {
+  NSString *artworkUrl = nil;
+
+  if (artwork) {
+      if ([artwork isKindOfClass:[NSString class]]) {
+           artworkUrl = artwork;
+      } else if ([[artwork valueForKey: @"uri"] isKindOfClass:[NSString class]]) {
```

Comment:
If you expect artwork to be a `NSString` type (from parameter type) then we won't ever pass in this condition. Right ?

Reply:
I'm not obj-c expert. But it works